### ISSUE TITLE
Define a struct for the PPU, APU, and CPU.

### ISSUE THREAD

**Issue by TrevorSpear** [vandomej/PandEm#8]

**Comment by vandomej** [comment]
Documentation for these can be found here:
    PPU: http://wiki.nesdev.com/w/index.php/PPU
    APU: http://wiki.nesdev.com/w/index.php/APU
    CPU: http://wiki.nesdev.com/w/index.php/CPU

    Overall combined they make up the 2A03 chip: http://wiki.nesdev.com/w/index.php/2A03

**Comment by vandomej** [comment]
I think that we should generally do the GUI and the shell of the program first before we get deep into the hardware. What do you think?

**Comment by vandomej** [comment]
Because if we are working on the backend, we need a way to test it to see if its actually working, and for that I think we need the screen at least to be working on the GUI end.

**Comment by TrevorSpear** [comment]
I agree on that because I need to make a list of how that roms work with the emulator.

**Comment by vandomej** [comment]
sounds good

**Comment by TrevorSpear** [comment]
To do later
